src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -205,6 +212,8 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
                     timeout=None,
                     context_path=None,
                     commit_trigger_enabled=None,
+                    pull_request_trigger_enabled=None,
+                    source_trigger_name=None,
```

Comment:
remove unused var: `source_trigger_name`

Reply:
Thanks for catching this! See newest update.